spec/cordial/orders_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+RSpec.describe Cordial::Orders do
+  let(:order_id) { '33451' }
+  let(:email) { 'cordial@example.com' }
+  let(:purchase_date) { '2018-07-02 13:19:00' }
+  let(:items) do
+    [
+      {
+        productID: '1',
+        sku: '123',
+        name: 'Test Product 1',
+        attr: {
+          color: 'blue',
+          size: 'L'
+        }
+      }
+    ]
+  end
+
+  describe '#find' do
+    subject { described_class.find(id: order_id) }
+
+    it 'has a correctly formatted request url' do
+      expect(subject.request.last_uri.to_s).to eq 'https://api.cordial.io/v1/orders/33451'
+    end
+  end
+
+  describe '#create' do
+    subject do
+      described_class.create(id: order_id,
+                             email: email,
+                             purchase_date: purchase_date,
+                             items: items)
+    end
+
+    it 'has a correctly formatted request url' do
```

Comment:
I set us on a bad path with this naming convention for `it` blocks. When you look at the output of rspec it's not very helpful in describing what we're testing, because it's all just the same `has a correctly formatted request url`. Not blocking this PR, but open to suggestions moving forward.

Reply:
Should we remove this test from all the resources?

Maybe we could add tests using stubs to give more  information about the behavior

What do you think, can we open an issue to address this later?